ARM-wvd-templates/HCI/AddHciVirtualMachinesTemplate.json

```diff
@@ -0,0 +1,323 @@
+        },
+        "defaultValue": ""
+      },
+      "hostpoolLocation": {
+        "type": "string",
+        "metadata": {
+          "description": "The location of the host pool to be updated. Used when the host pool was created empty."
+        },
+        "defaultValue": ""
+      },
+      "hostpoolProperties": {
+        "type": "object",
+        "metadata": {
+          "description": "The properties of the Hostpool to be updated. Used when the host pool was created empty."
+        },
+        "defaultValue": {}
+      },
+      "vmAdministratorAccountUsername": {
+        "type": "string",
+        "metadata": {
+          "description": "A username to be used as the virtual machine administrator account. The vmAdministratorAccountUsername and  vmAdministratorAccountPassword parameters must both be provided. Otherwise, domain administrator credentials provided by domainAdministratorUsername and domainAdministratorPassword will be used."
+        },
+        "defaultValue": ""
+      },
+      "vmAdministratorAccountPassword": {
+        "type": "securestring",
+        "metadata": {
+          "description": "The password associated with the virtual machine administrator account. The vmAdministratorAccountUsername and  vmAdministratorAccountPassword parameters must both be provided. Otherwise, domain administrator credentials provided by domainAdministratorUsername and domainAdministratorPassword will be used."
+        },
+        "defaultValue": ""
+      },
+      "oUPath": {
+        "type": "string",
+        "metadata": {
+          "description": "OU Path for the domain join"
+        },
+        "defaultValue": ""
+      },
+      "domain": {
+        "type": "string",
+        "metadata": {
+          "description": "Domain to join"
+        },
+        "defaultValue": ""
+      },
+      "domainAdministratorUsername": {
+        "type": "string",
+        "metadata": {
+          "description": "A username in the domain that has privileges to join the session hosts to the domain. For example, 'vmjoiner@contoso.com'."
+        },
+        "defaultValue": ""
+      },
+      "domainAdministratorPassword": {
+        "type": "securestring",
+        "metadata": {
+          "description": "The password that corresponds to the existing domain username."
+        },
+        "defaultValue": ""
+      },
+      "vmResourceGroup": {
+        "type": "string",
+        "metadata": {
+          "description": "The resource group of the session host VMs."
+        }
+      },
+      "vmLocation": {
+        "type": "string",
+        "metadata": {
+          "description": "The location of the session host VMs."
+        }
+      },
+      "customLocationId": {
+        "type": "string",
+        "metadata": {
+          "description": "A deployment target created and customized by your organization for creating virtual machines. The custom location is associated to an Azure Stack HCI cluster."
```

Comment:
I just looked this up, and this needs to be the full ARM path to something like:  /subscriptions/a95612cb-f1fa-4daa-a4fd-272844fa512c/resourceGroups/dogfoodarc/providers/Microsoft.ExtendedLocation/customLocations/dogfood-location .    Can we update the description to be clear this is the full arm path ?